scripts/src/errors.ts

```diff
@@ -0,0 +1,157 @@
+export type ApiError = {
+	code: number;
+	error: string;
+	message: string;
+};
+
+/**
+ * Code additions (/postfix) to be added to the http status code per error.
```

Comment:
Why the `/` before postfix? (also, although postfix can be used I think the more correct term is suffix...but I'm just being an Ory now...leave it or change it IDC)